ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/disk-encryption-set/provision/cucushift-installer-rehearse-azure-ipi-disk-encryption-set-provision-chain.yaml

```diff
@@ -7,7 +7,8 @@ chain:
   - ref: ipi-conf-telemetry
   - ref: ipi-conf-azure
   - ref: ipi-conf-azure-provisioned-des
-  - ref: ipi-install-monitoringpvc
+  #https://issues.redhat.com/browse/OCPBUGS-984
```

Comment:
There is an post step to enable monitor pvc - `ipi-install-post-monitoringpvc`, I think we can add `ipi-install-post-monitoringpvc` after `ipi-install` chain.

Reply:
good to know, change to use this step.